Codes/Brawlback/Netplay.cpp

```diff
@@ -11,13 +11,13 @@ namespace Netplay {
     bool IsInMatch() { return isInMatch; }
     void SetIsInMatch(bool isMatch) { isInMatch = isMatch; }
 
-    GameSettings gameSettings = GameSettings();
-    GameSettings* getGameSettings() { return &gameSettings; }
+    GameSettingsImpl gameSettings = GameSettingsImpl();
+    GameSettingsImpl* getGameSettings() { return &gameSettings; }
```

Comment:
why does this getGameSettings function exist? Can't we just get the game settings using the global `gameSettings` variable directly?